gobblin-docs/user-guide/Gobblin-Schedulers.md

```diff
@@ -44,7 +44,7 @@ This guide focuses on getting Gobblin to run in as a stand alone Java Process. T
 
 `gobblin-oozie-example-workflow.properties` contains default Oozie properties for any job launched. It is also the entry point for launching an Oozie job (e.g. to launch an Oozie job from the command line you execute `oozie job -config gobblin-oozie-example-workflow.properties -run`). In this file one needs to update the `name.node` and `resource.manager` to the values specific to their environment. Another important property in this file is `oozie.wf.application.path`; it points to a folder on HDFS that contains any workflows to be run. It is important to note, that the `workflow.xml` files must be on HDFS in order for Oozie to pick them up (this is because Oozie typically runs on a separate machine as any client process).
 
-`gobblin-oozie-example-workflow.xml` contains an example Oozie workflow. This example simply launches a Java process that invokes the main method of the [`CliLocalJobLauncher`](https://github.com/linkedin/gobblin/blob/master/gobblin-runtime/src/main/java/gobblin/runtime/local/CliLocalJobLauncher.java). The main method of this class expects two file paths to be passed to it (once again these files need to be on HDFS). The `jobconfig` arg should point to a file on HDFS containing all job configuration parameters. An example `jobconfig` file can be found [here](https://github.com/linkedin/gobblin/blob/master/gobblin-example/src/main/resources/wikipedia.pull). The `sysconfig` arg should point to a file on HDFS containing all system configuration parameters. An example `sysconfig` file for Oozie can be found [here](https://github.com/linkedin/gobblin/blob/master/gobblin-oozie/src/main/resources/local/gobblin-oozie-example-system.properties).
+`gobblin-oozie-example-workflow.xml` contains an example Oozie workflow. This example simply launches a Java process that invokes the main method of the [`CliLocalJobLauncher`](https://github.com/apache/incubator-gobblin/blob/master/gobblin-runtime/src/main/java/org/apache/gobblin/runtime/local/CliLocalJobLauncher.java). The main method of this class expects two file paths to be passed to it (once again these files need to be on HDFS). The `jobconfig` arg should point to a file on HDFS containing all job configuration parameters. An example `jobconfig` file can be found [here](https://github.com/linkedin/gobblin/blob/master/gobblin-example/src/main/resources/wikipedia.pull). The `sysconfig` arg should point to a file on HDFS containing all system configuration parameters. An example `sysconfig` file for Oozie can be found [here](https://github.com/linkedin/gobblin/blob/master/gobblin-oozie/src/main/resources/local/gobblin-oozie-example-system.properties).
```

Comment:
the Oozie config link should be changed to: https://github.com/apache/incubator-gobblin/blob/master/gobblin-oozie/src/test/resources/local/gobblin-oozie-example-system.properties